OpenRA.Mods.Common/Traits/World/TeamTogether.cs

```diff
@@ -0,0 +1,63 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Controls the team together checkbox in the lobby options.")]
+	public class TeamTogetherInfo : TraitInfo, ILobbyOptions
+	{
+		[Desc("Descriptive label for the command allied units checkbox in the lobby.")]
+		public readonly string CmdAlliedUnitsCheckboxLabel = "Command Allied Units";
```

Comment:
```suggestion
		public readonly string CommandAlliedUnitsCheckboxLabel = "Command Allied Units";
```
etc. as your needless abbreviation in this large string makes no sense.

Reply:
done with e0cef99